plugins/collaajUpdatePlugin/collaajClientUpdate.ini

```diff
@@ -0,0 +1,12 @@
+[4_7]

```

Comment:
If this is an example file, then it should be named collaajClientUpdate.ini.template